intel-retail/rtsf-at-checkout-reference-design
# feat(loss detector): migrate loss-detector to App SDK 3.0.1

## PR Checklist



## What are you changing?

Migrating loss_detector to EdgeX App SDK 3.0.1
## Issue this PR will close

close: #**77**
closes: #77 
## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

- Run `make loss-detector`
- Comment out lines 27 - 82 in `docker-compose.loss-detection.yml`
- Run `make run-base`
- Optional: Run `make run-portainer` to check logs of loss-detector container
- Optional: Run `docker compose -f loss-detection-app/docker-compose.portainer.yml down` to close Portainer

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/rtsf-at-checkout-reference-design )
